code/controllers/subsystem/vote.dm

```diff
@@ -131,59 +138,64 @@ SUBSYSTEM_DEF(vote)
 					restart = TRUE
 			if("gamemode")
 				if(GLOB.master_mode != .)
-					SSticker.save_mode(.)
-					if(SSticker.HasRoundStarted())
-						restart = TRUE
-					else
+					if(!SSticker.HasRoundStarted())
 						GLOB.master_mode = .
```

Comment:
Why yeet this?

Reply:
I can reset this, there was a big where ```SSticker.save_mode(.)``` was setting the server's default gamemode once game mode votes were performed. if this occurs after I will PR this change in